discovery_engine/lib/src/domain/assets/asset.dart

```diff
@@ -64,6 +64,7 @@ class Asset {
 
 // Type of an asset.
 enum AssetType {
+  smbertConfig,
   smbertVocab,
   smbertModel,
```

Comment:
regarding the dart side, it would be nice to also just have a single `AssetType.smbert` variant. but that would require refactoring the whole data provider and asset copier code, which would be out of scope of this pr. should we add a ticket to the backlog @biernacki ?

Reply:
I think it's not so important to change that, so I wouldn't add a ticket for now.